.azure/gpu-tests-lite.yml

```diff
@@ -29,9 +30,8 @@ pr:
       - "src/lightning_lite/**"
       - "tests/tests_lite/**"
       - "setup.cfg"  # includes pytest config
-      - ".actions/**"
     exclude:
-      - "requirements/lite/docs.txt"
```

Comment:
Only the lite requirements were added in the include section above, so only the lite docs.txt file needs to be excluded

Reply:
how about deep structure? https://github.com/Lightning-AI/lightning/pull/15770#discussion_r1029681627

Reply:
ok, keeping one-level and if there is only one then set the name...